executor/brie.go

```diff
@@ -46,6 +48,12 @@ import (
 	"github.com/pingcap/tidb/util/sqlexec"
 )
 
+const (
+	defaultCapOfCreateTable    = 512
+	defaultCapOfCreateDatabase = 64
+	brViaSQLComment            = `/*from(br_via_sql)*/`
```

Comment:
Should we distinguish `br via sql` with `br` https://github.com/pingcap/br/blob/master/pkg/gluetidb/glue.go#L30?
Or we can set this to `/*from(br)*/` since we will merge br into tidb in the future.

Reply:
`/*from(br)*/` +1